src/opir.nims

```diff
@@ -1 +1,3 @@
+when defined(windows):
+  switch("passL", "-L\"C:\\Program Files\\LLVM\\lib\"")
```

Comment:
I think if we get `Program Files` folder, for instance would it be better as follows ?,

```nim
let LLVMPath = os.joinPath( os.getEnv("ProgramFiles"),"LLVM","lib" )
switch("passL", "-L" & LLVMPath )
```